### PR TITLE
Added syrianmovies ontology files

### PR DESCRIPTION
**Pull request by baraa722**

### CODE REVIEW

File: syrian-cinema-ontology/SyrianMoviesOntology.rdf
Comment (davidlehn) [inline syrian-cinema-ontology/SyrianMoviesOntology.rdf:1]: This service does not host data.  You must remove this file (and remove it from git history).  Host it somewhere else and use redirects here.  (For instance, many people host files on github pages.)

File: syrian-cinema-ontology/SyrianMoviesOntology.properties
Comment (davidlehn) [inline syrian-cinema-ontology/SyrianMoviesOntology.properties:1]: This service does not host data. You must remove this file (and remove it from git history). This looks like a mistake commit?

File: syrian-cinema-ontology/.htaccess [hunk 1]
```diff
@@ -0,0 +1,95 @@
+//Rewrite to www
```
Comment (davidlehn) [inline syrian-cinema-ontology/.htaccess:1]: Use correct comment syntax.

File: syrian-cinema-ontology/.htaccess [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Options +FollowSymLinks
+RewriteEngine on
+RewriteCond %{HTTP_HOST} ^https://github.com/baraa722/Syrian_Movies_Ontology[nc]
+RewriteRule ^(.*)$ http://www.https://github.com/baraa722/Syrian_Movies_Ontology/$1 [r=301,nc]
+
+//Block bad bots
```
Comment (davidlehn) [inline syrian-cinema-ontology/.htaccess:7]: Remove all this.  Filter bots on your site.

File: syrian-cinema-ontology/.htaccess [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+RewriteCond %{HTTP_USER_AGENT} ^WWWOFFLE [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Xaldon\ WebSpider [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Zeus
+RewriteRule ^.* - [F,L]
+
+//Change default directory page
+DirectoryIndex https://github.com/baraa722/Syrian_Movies_Ontology
+
+//Prevent viewing of .htaccess file
+<Files .htaccess>
```
Comment (davidlehn) [inline syrian-cinema-ontology/.htaccess:91]: Remove. The server config will do this for you.

File: syrian-cinema-ontology/Readme.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Contributions to this ontology are welcome. If you have suggestions for improvements, new classes, or additional properties, please open an issue or submit a pull request.
+
+## License
+
+This ontology is licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).
+
+## Contact
+
+For questions or feedback about this ontology, please contact [Your Name or Organization](mailto:your.email@example.com).
```
Comment (davidlehn) [inline syrian-cinema-ontology/Readme.md:33]: Fix.  And add github ids for the maintainers.

**Review by davidlehn — CHANGES_REQUESTED** [review]
There are too many problems here to review further.  Please look at how other directories are setup, do something similar, and host your files elsewhere.  You will need to squash away the data files from the PR.  If you don't know how to do that, I'd suggest closing this and starting over.

**Comment by baraa722** [comment]
Sorry if I caused problems, i shall remove the entire fork.

**Pull request closed without merging**